Overview.src.html

```diff
@@ -949,6 +971,34 @@ <h4 id=terminology-headers>Headers</h4>
 Unless stated otherwise, it is unset.
 
 <p>A <span title=concept-request>request</span> has an associated
+<dfn title=concept-request-use-token-binding>use-token-binding flag</dfn>.
+Unless stated otherwise, it is unset.
+
+<p class="note no-backref"> <span title=concept-request>Request</span>'s
+<span title=concept-request-use-token-binding>use-token-binding</span> flag indicates whether the
+user agent will send its <span title=concept-token-binding-id>Token Binding Id</span>
+for the <span title=concept-request-origin>origin</span> along with the
+<span title=concept-request>request</span> when it transmits the
+<span title=concept-request>request</span> to the server.
+
+<p>A <span title=concept-request>request</span> has an associated
+<dfn title=concept-request-use-referred-token-binding>use-referred-token-binding flag</dfn>.
+Unless stated otherwise, it is unset.
+
+<p class="note no-backref"> <span title=concept-request>Request</span>'s
+<span title=concept-request-use-referred-token-binding>use-referred-token-binding</span>
+flag controls whether the user agent will send the
+<span title=concept-token-binding-id>Token Binding Id</span>
+used by the user agent with the referring <span title=concept-request-origin>origin</span>,
+in addition to the <span title=concept-token-binding-id>Token Binding Id</span> used by the
+user agent with the <span title=concept-request-origin>origin</span>, when it transmits the
+<span title=concept-request>request</span> to the server.
+
+<p>A <span title=concept-request>request</span> has an associated
+<dfn title=concept-request-referred-token-binding-origin>referred-token-binding-origin</dfn>,
+which is an <span title=concept-request-origin>origin</span>. Unless stated otherwise, it is unset.
```

Comment:
This should take either null or an origin with null as the default. Only flags can be unset.